recipe/build.sh

```diff
@@ -1,19 +1,17 @@
 #!/bin/bash
 
-# default is 10.5 for some reason, which doesn't work
+EXTRA_CMAKE_ARGS=""
 if [[ `uname` == 'Darwin' ]];
 then
-    export MACOSX_DEPLOYMENT_TARGET=10.7
-    export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+    export CMAKE_OSX_DEPLOYMENT_TARGET=""
```

Comment:
Why is this being unset? If anything, we should start setting this to `MACOSX_DEPLOYMENT_TARGET`, which is [set in `conda-forge-build-setup`]( https://github.com/conda-forge/conda-forge-build-setup-feedstock/blob/2a2a0ba2187aa16f972b9b4e2af4c76dbf0aa386/recipe/run_conda_forge_build_setup_osx#L9 ).